plugins/mcreator-core/datalists/blocksitems.yaml

```diff
@@ -1986,6 +2003,7 @@
 - Blocks.MANGROVE_SIGN:
   readable_name: "Mangrove Sign"
   type: block
+  texture: MANGROVE_SIGN

```

Comment:
This doesn't match the other standing sign textures

Reply:
Nice catch!